cmd/cartesi-rollups-node/services.go

```diff
@@ -310,3 +311,10 @@ func newStateServer() services.Service {
 	s.Env = append(s.Env, os.Environ()...)
 	return s
 }
+
+func newHealthService() services.Service {
```

Comment:
The health check shouldn't be a service on its own. It should be part of an HTTP service that will have multiple endpoints. One of them is the health check. The proxy defines the other endpoints. For inspiration, see https://github.com/gligneul/nonodo/blob/main/internal/nonodo/echo.go.